host/xtest/xtest_test.c

```diff
@@ -196,3 +197,7 @@ const TEEC_UUID storage_benchmark_ta_uuid = TA_STORAGE_BENCHMARK_UUID;
 const TEEC_UUID socket_ta_uuid = TA_SOCKET_UUID;
 const TEEC_UUID sdp_basic_ta_uuid = TA_SDP_BASIC_UUID;
 const TEEC_UUID tpm_log_test_ta_uuid = TA_TPM_LOG_TEST_UUID;
+const TEEC_UUID ocall_ta_uuid = TA_OCALL_UUID;
+#ifdef WITH_GP_TESTS
+const TEEC_UUID gp_tta_ds_uuid = TA_TTA_DS_UUID;
+#endif
```

Comment:
Rebase error, was removed by commit c1acd29af6b0 ("xtest: remove GP test TA UUID")

Reply:
Removed, thank you!